Sources/PushNotifications.swift

```diff
@@ -364,9 +364,21 @@ import Foundation
     }
 
     private func syncInterests() {
-        // Sync saved interests when app starts.
-        guard let interests = self.getInterests() else { return }
-        try? self.setSubscriptions(interests: interests)
+        // Sync saved interests when app starts, if necessary.
+        guard
+            let interests = self.getInterests(),
+            let interestsHash = interests.calculateMD5Hash(),
```

Comment:
I think that if we fail to calculate the hash for some reason, we still want the requests to fire.